.github/workflows/markdown-diff-lint.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: Lint Markdown file changes
```

Comment:
Sorry if this is a naive question - just want to make sure we have thought about it. Should we implement this with prow or GitHub actions?

Reply:
Good point. I'm fine either way. If we decide to go with prow, there are a couple of things that I'm not sure about:

* Is this repo already active in prow? Can I drop a new configuration file, or must we request it be activated?
* There are some required variables that I don't know the equivalent in the prow environment. I'm sure they must be available, but I'm still learning prow :), namely:
   * BASE_REF: github.event.pull_request.base.ref
   * BASE_CLONE_URL: github.event.pull_request.base.repo.clone_url
   * HEAD_CLONE_URL: github.event.pull_request.head.repo.clone_url

To ensure that we are on the same page (and dumping my thoughts so I don't forget ;), we'll need to use a node base image as release-notes does. Refer to their [release-notes-presubmits.yaml](https://github.com/kubernetes/test-infra/blob/73346eb2d68449fb9fec028c0547022fdfa0356d/config/jobs/kubernetes-sigs/release-notes/release-notes-presubmits.yaml#L9-L15).

Reply:
Let's try and go to prow in the first instance. I don't think we have any existing jobs for this repo but I don't see why it wouldn't be possible.  We will need to customise https://github.com/kubernetes/test-infra/blob/master/config/jobs/etcd/etcd-periodics.yaml#L8-L11 for example to refer to `website` rather than `etcd`.

Let's refine this pr to just be the scripting / Makefile / markdownlint config and raise a follow-up in test-infra to attempt adding a prow job.

Reply:
I made the changes to make the script standalone. Can you PTAL? Thanks